react/InstallmentsList.tsx

```diff
@@ -40,9 +32,9 @@ function InstallmentsList(props: Props) {
 
   return (
     <div className={handles.installmentsListContainer}>
-      {pickedInstallments.map((inst: Installments, i: number) => {
+      {pickedInstallments.map((inst, i: number) => {
         return (
-          <InstallmentsItem installments={inst} key={i}>
+          <InstallmentsItem installment={inst} key={i}>
```

Comment:
Also think changing this from `installments` to `installment` is not a good idea